R/layouts.R

```diff
@@ -241,3 +241,69 @@ split_layout <- function(..., cell_widths = NULL, cell_args = "", style = NULL){
 splitLayout <- function(..., cellWidths = NULL, cellArgs = "", style = NULL) {
   split_layout(..., cell_widths = cellWidths, cell_args = cellArgs, style = style)
 }
+
+
+#' Vertical layout
+#'
+#' Lays out elements vertically, one by one below one another.
+#'
+#' @param ... Unnamed arguments will become child elements of the layout.
+#' @param rows_heights Character or numeric vector indicating the widths of the
+#' individual cells. Recycling will be used if needed.
+#' @param cell_args character with additional attributes that should be used for
+#' each cell of the layout.
+#' @param adjusted_to_page if TRUE it adjust elements position in equal spaces to
+#' the size of the page
+#' @param fluid not supported yet (here for consistency with \code{shiny})
+#'
+#' @return vertical layout grid object
+#' @export
+#' @rdname vertical_layout
+#' @examples
+#' if (interactive()) {
+#'   ui <- semanticPage(
+#'     verticalLayout(
+#'       a(href="http://example.com/link1", "Link One"),
+#'       a(href="http://example.com/link2", "Link Two"),
+#'       a(href="http://example.com/link3", "Link Three")
+#'     )
+#'   )
+#'   shinyApp(ui, server = function(input, output) { })
+#' }
+#' if (interactive()) {
+#'   ui <- semanticPage(
+#'     vertical_layout(h1("Title"), h4("Subtitle"), p("paragraph"), h3("footer"))
+#'   )
+#'   shinyApp(ui, server = function(input, output) { })
+#' }
```

Comment:
Good job! It'd be great to have some simple app in examples folder

Reply:
added